test/e2e-tests.sh

```diff
@@ -82,6 +82,43 @@ function test_setup() {
   $(dirname $0)/upload-test-images.sh e2e || fail_test "Error uploading test images"
 }
 
+readonly PUBSUB_SERVICE_ACCOUNT="eventing_pubsub_test"
+readonly PROJECT_ID="${GCP_PROJECT}"
+readonly PUBSUB_SERVICE_ACCOUNT_KEY="knative-gcppubsub-channel.json"
```

Comment:
This should be a temp file.

Reply:
Changed to `readonly PUBSUB_SECRET_NAME="gcppubsub-channel-key"`